templates/cfp/form_installation_include.html

```diff
@@ -1,6 +1,14 @@
 <h3>What we're looking for</h3>
 <p>We're looking for installations for our attendees to admire and interact with all over the site. Think big.</p>
 <p>In previous years we've had towers of flame, robots to drag around, trees that react to breathing, visualisations of black holes, fields of light that swarm around you, and domes as far as the eye can see.</p>
+<p>Please fill in this form if any of the following are true of your (proposed) installation:</p>
+<ul>
+  <li>Safety concerns &mdash; if it involves fire, high power lasers, whirling blades etc.</li>
+  <li>Specific siting &mdash; does it need a lot of space, is it very noisy, want it somewhere not in your tent or village or do you want to place it in Null Sector?</li>
```

Comment:
This is a bit hard to read. Maybe:

Location &mdash; does it need a lot of space or is it very noisy? Is it best placed outside your village, or do you want it to be part of Null Sector?